infrastructure/models.py

```diff
@@ -400,6 +433,31 @@ def fuzzy_planned_completion_date(self):
             self.planned_completion_month,
             self.planned_completion_day
         )
+
+    @property
+    def pipeline_capacity_property(self):
+        if self.project_capacity is None:
+            return None
+        pc = str(self.project_capacity)
+        if self.project_capacity_unit:
+            pc += " {}".format(ProjectCapacityUnits.get_human(self.project_capacity_unit))
```

Comment:
I think we should lower the casing. Currently we show **3.8 Tons per year** as opposed to **3.8 tons per year** (which is in the PDF)

Reply:
This is another case of "this is the format in the pick list that they gave us" so I thought best to follow it. I agree, lower case looks better. If we do lower case, should we lower case only in the template view, but  keep the upper case in the admin?